src/ontology/mondo-edit.obo

```diff
@@ -104995,20 +104995,22 @@ is_a: MONDO:0005560 {source="DOID:4267", source="EFO:0007138", source="MESH:D000
 id: MONDO:0005641
 name: aleutian mink disease
 def: "A slow progressive disease of mink caused by the aleutian mink disease virus. It is characterized by poor reproduction, weight loss, autoimmunity, hypergammaglobulinemia, increased susceptibility to bacterial infections, and death from renal failure. The disease occurs in all color types, but mink which are homozygous recessive for the Aleutian gene for light coat color are particularly susceptible." [MESH:D000453]
-subset: otar {source="MONDO:OTAR"}
-synonym: "Aleutian disease" EXACT [DOID:2934]
-synonym: "Aleutian mink disease virus caused disease or disorder" EXACT [MONDO:patterns/specific_infectious_disease_by_agent]
-synonym: "Aleutian mink disease virus disease or disorder" EXACT []
-synonym: "Aleutian mink disease virus infectious disease" EXACT []
-xref: DOID:2934 {source="EFO:0007139", source="MONDO:equivalentTo"}
-xref: EFO:0007139 {source="MONDO:equivalentTo", source="MONDO:EFO"}
+comment: In Mondo we make the distinction between human and non-human animal diseases. Cross-references do not necessarily make this distinction. Annotate to the human or non-human animal term as appropriate. This disease is considered zoonotic. {source="PMID:19961696"}
+synonym: "Aleutian disease" RELATED [DOID:2934]
+synonym: "Aleutian mink disease virus caused disease or disorder" RELATED [MONDO:patterns/specific_infectious_disease_by_agent]
+synonym: "Aleutian mink disease virus disease or disorder" RELATED []
+synonym: "Aleutian mink disease virus infectious disease" RELATED []
+xref: DOID:2934 {source="EFO:0007139", source="MONDO:relatedTo"}
+xref: EFO:0007139 {source="MONDO:relatedTo", source="MONDO:EFO"}
 xref: ICD9:079.89 {source="MONDO:relatedTo", source="MONDO:i2s"}
-xref: MESH:D000453 {source="EFO:0007139", source="MONDO:equivalentTo", source="DOID:2934"}
-xref: SCTID:62251004 {source="MONDO:equivalentTo", source="DOID:2934"}
-xref: UMLS:C0002016 {source="MONDO:equivalentTo", source="DOID:2934"}
-is_a: MONDO:0005108 {source="DOID:2934", source="EFO:0007139", source="MESH:D000453/inferred"} ! viral infectious disease
-intersection_of: MONDO:0005550 ! infectious disease
+xref: MESH:D000453 {source="EFO:0007139", source="MONDO:relatedTo", source="DOID:2934"}
+xref: SCTID:62251004 {source="MONDO:relatedTo", source="DOID:2934"}
+xref: UMLS:C0002016 {source="MONDO:relatedTo", source="DOID:2934"}
+is_a: MONDO:0700206 {source="DOID:2934", source="EFO:0007139", source="MESH:D000453/inferred"} ! Parvoviridae infectious disease, non-human animal
+intersection_of: MONDO:0700049 ! infectious disease, non-human animal
 intersection_of: MONDO:0100332 NCBITaxon:28314 ! disease has primary infectious agent Aleutian mink disease parvovirus
+relationship: in_taxon NCBITaxon:169418 ! Mustelinae
+relationship: MONDO:0700097 MONDO:1010029 ! cross-species analog aleutian mink disease, human
```

Comment:
please add a source (you can also add your orcid)

Reply:
Done